build_blackbox_tests

```diff
@@ -35,13 +35,7 @@ done
 for D in tests/stubs/*; do
 	if [ -d "${D}" ]; then
 		echo "Building ${D}"
-		if [ "$GOHOSTARCH" == "$GOARCH" ]; then
-			go install ${REPO_PATH}/${D}
-		else
-			# Don't try to go install things when cross compiling because
-			# go install disallows it.
-			go build ${REPO_PATH}/${D}
-		fi
+		go build -ldflags "${GLDFLAGS}" -o ${BIN_PATH}/$(basename ${D}) ${REPO_PATH}/${D}
```

Comment:
I don't think we need to pass the `GLDFLAGS` but I don't think it'll hurt anything either.

Reply:
Oh right, those won't have any impact here. Updated.